test.lisp

```diff
@@ -123,6 +125,12 @@
       (setf (children parent) (remove test (children parent))))
     name))
 
+(defun remove-all-tests-in-package (&optional (package *package*))
+  "Remove all tests from the current PACKAGE (current by default)."
+  (let ((*package* (find-package package))) ; didn't work without this
```

Comment:
Instead of binding `*package*`, pass the package as the second argument to `remove-test`.